main.py

```diff
@@ -170,6 +184,10 @@ def _update_switch_flows(self, switch):
         switch.flows = self._multipart_replies_flows[switch.id]
         del self._multipart_replies_flows[switch.id]
         del self._multipart_replies_xids[switch.id]['flows']
+        event_raw = KytosEvent(
+            name='kytos/of_core.flow_stats.received',
```

Comment:
Is this really the right event to send here? Or should we use another event name?

Reply:
Hi Humberto! thank you for your comment! In fact, the place could be the caller function (_handle_multipart_flow_stats) which will be much more easy to understand